enterprise/backend/src/metabase_enterprise/audit_app/pages/common/cards.clj

```diff
@@ -33,6 +49,12 @@
                     :group-by [:card_id]}
                    (common/add-45-days-clause :started_at))])
 
+(def dashboards-count
+  "HoneySQL for a CTE to enumerate the dashboards for a Card."
+  [:dash_card {:select [:card_id [:%count.* :count]]
+               :from [:report_dashboardcard]
+               :group-by [:card_id]}])
+
```

Comment:
If all this stuff is only used in one query in one other namespace (`-queries`) does it make sense just to put it there? That way you don't have to flip around between two files to be able to understand what's going on in the `::bad-table` query... especially since these CTEs introduce table aliases that you have to know about to use them.

Just a thought. I'm not stressed about it either way